tools/travis/before_script.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+mkdir -p /root/.cache/unity3d
+mkdir -p /root/.local/share/unity3d/Unity/
```

Comment:
One mkdir ends without a trailing slash, the other with. I know that both do exactly the same, though I usually try to keep them consistent within the same file, especially if they're two consecutive lines.